projects/1.21-fabric/assets/sodium/sodium/lang/zh_cn.json

```
@@ -0,0 +1,69 @@
+{
+    "sodium.option_impact.low": "低",
+    "sodium.option_impact.medium": "中",
+    "sodium.option_impact.high": "高",
+    "sodium.option_impact.extreme": "极高",
+    "sodium.option_impact.varies": "视情况而定",
+    "sodium.options.pages.quality": "质量",
+    "sodium.options.pages.performance": "性能",
+    "sodium.options.pages.advanced": "高级",
+    "sodium.options.view_distance.tooltip": "渲染距离控制地形可视范围。较低值可减少地形渲染量以提高帧率。",
+    "sodium.options.simulation_distance.tooltip": "模拟距离控制地形与实体的加载范围。较低值可降低服务器负载并可能提升帧率。",
+    "sodium.options.brightness.tooltip": "控制世界的最低亮度。提高该值会使黑暗区域更明亮，但不会影响已充分照明的区域。",
+    "sodium.options.gui_scale.tooltip": "设置用户界面最大缩放比例。若使用“自动”，则始终采用最大可用比例。",
+    "sodium.options.fullscreen.tooltip": "启用后，游戏将以全屏模式运行（需硬件支持）。",
+    "sodium.options.fullscreen_resolution.tooltip": "全屏模式使用的显示器分辨率与刷新率。更改此选项可能干扰其他应用程序并导致切换延迟。\n\n仅支持Windows操作系统。",
+    "sodium.options.v_sync.tooltip": "启用后，游戏帧率将与显示器刷新率同步以获得更平滑画面，但可能增加输入延迟。硬件性能不足时可能导致帧率下降。",
+    "sodium.options.fps_limit.tooltip": "限制最大帧率。可降低多任务时的系统负载与电池消耗。若垂直同步已启用，则仅当此值低于刷新率时生效。",
+    "sodium.options.view_bobbing.tooltip": "启用后，玩家移动时视角会产生晃动效果。易晕眩玩家建议禁用。",
+    "sodium.options.attack_indicator.tooltip": "控制攻击指示器在屏幕上的显示位置。",
+    "sodium.options.autosave_indicator.tooltip": "启用后，游戏自动保存时将显示提示图标。",
+    "sodium.options.graphics_quality.tooltip": "默认画质控制部分传统选项并确保模组兼容性。下方选项设为“默认”时将继承此设置。",
+    "sodium.options.clouds_quality.tooltip": "控制天空云层的渲染质量。尽管选项标记为“流畅”与“高品质”，但对性能的影响微乎其微。",
+    "sodium.options.weather_quality.tooltip": "控制雨雪等天气特效的渲染距离。",
+    "sodium.options.leaves_quality.name": "树叶渲染",
+    "sodium.options.leaves_quality.tooltip": "控制树叶渲染为透明（流畅）或不透明（高品质）。",
```

写反啦，这里原文前面是高品质，后面才是流畅。
```suggestion
    "sodium.options.leaves_quality.tooltip": "控制树叶渲染为透明（高品质）或不透明（流畅）。",
```